front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/CraftSections/ImageTextCards/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+// components
+import { Box } from '@citizenlab/cl2-component-library';
+
+// craft
+import { UserComponent, Element } from '@craftjs/core';
+
+// widgets
+import TwoColumn from 'components/admin/ContentBuilder/Widgets/TwoColumn';
+import Container from 'components/admin/ContentBuilder/Widgets/Container';
+import ImageMultiloc from 'components/admin/ContentBuilder/Widgets/ImageMultiloc';
+import TextMultiloc from 'components/admin/ContentBuilder/Widgets/TextMultiloc';
+import WhiteSpace from 'components/admin/ContentBuilder/Widgets/WhiteSpace';
+
+const ImageTextCards: UserComponent = () => {
+  return (
+    <Element id="image-text-cards" is={Box} canvas>
+      <TwoColumn columnLayout="1-2">
+        <Element id="left" is={Container} canvas>
+          <ImageMultiloc alt={{}} />
```

Comment:
Maybe we can just make these attributes optional instead of giving them an empty object?